soweego/wikidata/api_requests.py

```diff
@@ -656,6 +676,6 @@ def _make_buckets(qids):
 
 if __name__ == "__main__":
```

Comment:
We should delete this `if`, probably a debug session leftover 😃

Reply:
this has been resolved in f985fb9231c19f4f842968becdfb83fc523c28f2